src/components/Form/formFunctions.ts

```diff
@@ -147,6 +148,22 @@ export const onChangeDateOrTime = (
     fieldSetter(key, value);
 };
 
+export const onChangeDate = (
```

Comment:
why do we need this function - instead of just put the additional logic (lines 158 to 164) as more if statements in `onChangeDateTime`?

Reply:
Because onChangeDate is only applied to the dates